### ISSUE TITLE
error TS2339: Property 'endsWith' does not exist on type 'string'.

### ISSUE THREAD

**Issue by rajinder-yadav** [Microsoft/TypeScript#12384]
tsc 2.0.10

**Code**

let tok = "path/";
if(tok.endsWith("/"))  <--- false error:error TS2339: Property 'endsWith' does not exist on type 'string'.
```ts
// A *self-contained* demonstration of the problem follows...

```

**Expected behavior:**
should know this is a method on a JS string.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith

**Actual behavior:**

**Comment by j-oliveras** [comment]
The method endsWith is declared with es6 (es2015). Make sure your target is es6 or higher (if target is not defined defaults to es3).

**Comment by j-oliveras** [comment]
The method endsWith is declared into es6 (es2015). Make sure your target is es6 or higher (if target is not defined defaults to es3) or use lib option to include "es2015.core".

**Comment by rajinder-yadav** [comment]
Thanks, closing!
